server/lib/nl/fulfillment/time_delta_across_vars.py

```diff
@@ -99,12 +100,15 @@ def populate(state: PopulateState, chart_vars: ChartVars, places: List[Place],
 
     # TODO: Uncomment this once we agree on look and feel.
     if field == 'abs':
+      sv_place_latest_date = get_sv_place_latest_date(ranked_svs, places, None,
+                                                      state.exist_checks)
```

Comment:
It slightly feels to me that the data-structure for this and get_sv_place_facet_ids could be unified, since they both come into play when there's date range, right?  Would that make sense?

I'm also wondering why this is not conditioned under date_range check?

Reply:
Feel free to defer to post submission :)

Reply:
actually, facet_ids is needed for timeline charts (both single date and date range) and place_latest_date is needed for single place charts like map and highlight and used for date range

& this is not conditioned under date_range check because get_sv_place_latest_date actually will return empty when there's no date range and there's a comment on the function explaining this